JavaAssignment/src/soumya/Addition.java

```diff
@@ -0,0 +1,17 @@
+package soumya;
+
+public class Addition {// Method for adding two integers
+	void add(int num1, int num2) {
+		System.out.println("Addition of two integer is: " + (num1 + num2));
+	}
+
+	// Method for adding two decimal
+	void add(double num1, double num2) {
+		System.out.println("Addition of two decimal is: " + (num1 + num2));
+	}
+
+	// Method for adding two string
+	void add(String str1, String str2) {
```

Comment:
Here need to parse String value to int and then you will add that number and return string value.